source/_integrations/medcom_ble.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Medcom Bluetooth
+description: Integrate International Medcom radiation monitors
+ha_category:
+  - Environment
+  - Sensor
+ha_release: '2023.10'
+ha_iot_class: Local Poll
+ha_codeowners:
+  - '@elafargue'
+ha_domain: medcom_ble
+ha_bluetooth: true
+ha_platforms:
+  - sensor
+ha_config_flow: true
+hw_integration_type: integration
+---
+
+Integrates International Medcom Bluetooth-enabled radiation monitors into Home Assistant.
+
+[International Medcom](https://medcom.com/) is an American company that manufactures high quality radiation detection instruments that are used in professional environments, for home and office, and by community projects worldwide.
```

Comment:
```suggestion
[International Medcom](https://medcom.com/) is an American company that manufactures radiation detection instruments that are used in professional environments, for home and office, and by community projects worldwide.
```


high quality seems like an endorsement, ideally we want docs to be a factual as possible